flexmeasures_openweathermap/utils/modeling.py

```diff
@@ -19,6 +19,16 @@ def get_or_create_owm_data_source() -> DataSource:
     )
 
 
+def get_or_create_owm_data_source_for_derived_data() -> DataSource:
+    owm_source_name = current_app.config.get(
+        "OPENWEATHERMAP_DATA_SOURCE_NAME", DEFAULT_DATA_SOURCE_NAME
+    )
+    return get_or_create_source(
+        source=f"FlexMeasures {owm_source_name}",
```

Comment:
In flexmeasures-entsoe we have a dedicated config setting for this, named `ENTSOE_DERIVED_DATA_SOURCE`. Perhaps we should streamline that.

Side note: in the longer run, I'd like to move towards better source lineage. The FlexMeasures model computing irradiance from cloud coverage combines two models (Ineichen 2002 and Perez 2007) with OWM data.

Reply:
It seems the design for derived sources is not settled yet. Also, I don't feel this is important enough to warrant a setting.